providers/os/resources/usb/parser_plist.go

```diff
@@ -0,0 +1,251 @@
+// Copyright (c) Mondoo, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package usb
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+type USBDevice struct {
```

Comment:
nit; `usb.Device` reads better 
```suggestion
type Device struct {
```